### PR TITLE
Improvements & crap

### PR DESCRIPTION
**Pull request by SK-Fast**
YE

lis of added crap: 
- quality selection
- folder selection
- queue
- modify ui a bit
- clear queue button
- :trollface:

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,43 +4,73 @@
...
-        <ins onclick="openVideosFolder()" class="hover:underline cursor-pointer text-green-500" id="done"></ins>
-        <code id="info"></code>
-        <code id="error"></code>
       </div>
     </div>
 
     <script src="js/downloader.js"></script>
+
+    <script>
+      const noConnection = document.getElementById("no-connection")
```
Comment (bluefield-creator) [inline src/index.html]: What's this?

File: src/js/downloader.js [hunk 2]
```diff
@@ -9,57 +9,243 @@ const fs = require("fs");
...
   }
 
   if (!fs.existsSync('./downloads'))
     fs.mkdirSync('./downloads');
 
-  const fileName = videoFileName ? videoFileName : nanoid(20);
-  videoName.textContent = fileName;
+  let data
+
+  try {
```
Comment (bluefield-creator) [inline src/js/downloader.js]: Why not do it inline?

```js
await ytdl.getBasicnInfo(videoURL).catch((err) => logEror(err.message))
```

File: src/js/downloader.js [hunk 3]
```diff
@@ -9,57 +9,243 @@ const fs = require("fs");
...
+    updateQueueCount()
+  } else {
+
+    downloadProgressElementTwo.textContent = "100%";
+    downloadProgressElement.style.width = "100%";
+    downloadProgressElement.style.backgroundColor = "#43e058"
+
+    updateQueueCount()
+    
+    /*
```
Comment (bluefield-creator) [inline src/js/downloader.js]: remove if commented

**Review by bluefield-creator — COMMENTED** [review]
Please use the smae package manager

File: src/index.html (hunk 1 above)
Comment (SK-Fast) [reply src/index.html]: Idk i planned to add these crap, forgo to remov

**Comment by SK-Fast** [comment]
Phickeddd ig

**Comment by bluefield-creator** [comment]
Before merguing, please include a screenshot of UI changes, and remove package-lock.json (only keep yarn.lock)

**Comment by bluefield-creator** [comment]
Sorry for late responses, Im still in school.

File: src/style/style.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif !important;
+    user-select: none;
```
Comment (bluefield-creator) [inline src/style/style.css]: user-select has its own class within tailwindcss called `select-none`. I'd prefer if it was used.

**Comment by SK-Fast** [comment]
Don

**Pull request merged**